### DISCUSSION TITLE
Networking API

### DISCUSSION THREAD

**Discussion by georgelam0306** [nxrighthere/UnrealCLR#89] [Help] [answered]
Is there an equivalent way of setting up replication as this doc: https://docs.unrealengine.com/en-US/InteractiveExperiences/Networking/Actors/RPCs/index.html?

I haven't been able to find anything related to setting up ue4's networking

**Comment by nxrighthere** [comment, marked as answer]
This is work-in-progress, you can track it in https://github.com/nxrighthere/UnrealCLR/issues/38
